test/e2e/storage/testsuites/volumeperf.go

```diff
@@ -105,6 +105,12 @@ func (t *volumePerformanceTestSuite) DefineTests(driver storageframework.TestDri
 		dInfo *storageframework.DriverInfo
 		l     *local
 	)
+
+	cleanup := func(ctx context.Context) { 
```

Comment:
How about adding a nil check to `AfterEach` instead? There's more cleanup there that would encounter similar problems.